java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -1,15 +1,14 @@
 package jmri.jmrit.display.layoutEditor;
 
-import static jmri.jmrit.display.layoutEditor.LayoutTrack.NONE;
 import static jmri.util.MathUtil.add;
```

Comment:
I blame netBeans: I'm just using the Source -> "Fix Imports" menu item.
I've noticed that sometimes it will import just the class… other times a specific method of the class… I'm not a Java-head… but if I'm understanding "import static <method>" it means I don't have to prefix the methods from that class with the class name… So I suspect that's what's happening is that in some source files I used the whole class+method name and in others I just used the method name… and netBeans is fixing the imports accordingly. So in the files that are doing the "import static <class.method>" I need to find the places I use those methods and add the class prefix. BRB… ;-)
(Thank you for your review; the noob appreciates it. ;-)